lib/init.go

```diff
@@ -0,0 +1,9 @@
+package lib
+
+var bogonIP4List []IPRange
+var bogonIP6List []IP6Range
```

Comment:
Since these are clearly bogon-related data structures, we can move them to `bogon.go` and make a note above them as such:

```go
// these lists are initialized on startup inside this pkg's `init`.
var bogonIP4List []IPRange
var bogonIP6List []IP6Range
```